src/sentry/conf/types/sdk_config.py

```diff
@@ -3,6 +3,10 @@
 from collections.abc import Callable
 from typing import Any, Literal, NotRequired, TypedDict
 
+# TODO(epurkhiser): This should be imported from the SDK, but it doesn't appear
+# to be exported?
+Event = Any
```

Comment:
I believe this should be re-exported in `sentry_sdk.typing.Event`.

Reply:
it's `sentry_sdk._types.Event` which is underscored so a no-no for relying on stably

Reply:
@asottile-sentry We have explicitly [re-exported the `Event` type in `sentry_sdk.types.Event`](https://github.com/getsentry/sentry-python/blob/master/sentry_sdk/types.py) (without the underscore) to indicate that it is part of the public API.